openreview/arr/webfield/programChairsWebfield.js

```diff
@@ -169,6 +169,62 @@ return {
       return checklistReplies?.length??0;
       `
     },
+    reviewerEmailFuncs: [
+      {
+        label: 'Available Reviewers with No Assignments', filterFunc: `
+        if (row.notesInfo.length > 0){
+          return false;
+        }
+
+        const registrationNotes = row.reviewerProfile?.registrationNotes ?? []
+        if (registrationNotes.length <= 0) {
+          return false
+        }
+
+        const maxLoadForm = registrationNotes.filter(note => {
+          const invitations = note?.invitations ?? []
+          return invitations.some(inv => inv.includes('Reviewers/-/Max_Load_And_Unavailability_Request'))
```

Comment:
for future functions, no need to check all invitations, checking the first of the array should be enough. 

The first invitation is the one that is being used to create the note.